README.md

```diff
@@ -2638,6 +2639,25 @@ if (!result.success) {
   // => ["Expected string, received number"]
 }
 ```
+### Using Zod with forms
+
+Zod and HTML forms share some common terms, but they do not mean the same thing, which can lead to some confusion.
+One common misstep is that the schema `z.string()` will allow empty strings for a required HTML form field, which matches the typescript type system, but might not be what you expect.
+```
```

Comment:
It's hard to use GitHub's suggestion feature on code blocks, but can you make the following changes:

1. Use ` ```ts`
2. The options you list below should also use ` ```ts` code fences in the examples.
3. Can you format the code examples like prettier with no configuration would? Our formatting is pretty inconsistent, but I'd like to improve that in general.